en/drafts/originals/space-place-gazetteers.md

```diff
@@ -25,122 +25,111 @@ doi: XX.XXXXX/phen0000
+The first task is to create a spreadsheet and determine the fields that you will populate with data taken from the historical text. Open Excel, or your preferred spreadsheet software. 
 
-Based on our source material, including a controlled vocabulary for the type of place is smart for the dataset. The Linked Places format recommends the use of a Place Type. Include a column called "PlaceType."
+In a widely cited 2006 book, the geospatial librarian Linda Hill suggested that each entry in a well-structured gazetteer should include at least one name, at least one set of coordinates, and one or more feature types.[^11] For historians, it is especially important to include modern place names if the name has changed, as well as the temporal range of the older name(s) attested in a source. With multilingual sources or projects, it may also be important to note different names or transliterations for a given place: for example, Moscow (EN), Moskau (DE), Moscou (FR), Москва (RU). In keeping with good practices for creating spreadsheets that may need to be shared or exported into other software, you should always include an ID number column. Start by naming the first column **ID**.
 
-Using an established standard like this means that the data you create in this tutorial, or that you create for your own research informed by this tutorial, can be shared with other likeminded researchers to create new knowledge. You will thus also include a column for aat_type, another strongly recommended standardized form of attribute data that makes it easier to share historical spatial project data. Type in "aat_type" for one of the columns. 
+From the first paragraph of *The Itinerary*, we understand that we need a column for the place names of travel stops: call the next column **TravelStop**. You could use whatever column headers you want for your own research, but the LP format recommends the use of a 'place type', so add a column called **PlaceType**. You will also include a column for 'attribute type', another strongly recommended standardized form of attribute data that makes it easier to share historical spatial project data. Type in **aat_type** in the next column. 
 
-As the second research question for this tutorial ties into questions of Jewish history, you should add a column for whether there is a Jewish population in the place. You should also add an additional column that includes further information about the description of the Jewish population. Call this column, "JewishPop." 
+Since the research question for this lesson also revolves around Jewish history, you should add a column to mark whether the place has a Jewish population. Call this column **JewishPop**. You might want to add a column that allows you to include further descriptions of the Jewish population, if there are any.
```

Comment:
I wondered what the "research question" was supposed to refer to exactly. Is it the sentence at line 112 (green): "A spatial historian interested in this text may want to discover where Benjamin of Tudela traveled on his grand journey, and how he interacted with Jewish communities in the locations he visited"?
If that's the case, I think it could be stated more explicitly that this is a research question that we will actively explore in our example case study.

Reply:
Yes that is the major research question for this example. If you want to add a line somewhere appropriate that says something along the lines of, "and this is the research questions that we will actively explore in our example case study," that's fine.